instructionAPI/src/Instruction.C

```diff
@@ -558,13 +558,24 @@ namespace Dyninst
                 case e_sysexit:
                 case e_call:
                 case e_syscall:
-                case amdgpu_op_s_setpc_b64:
-                case amdgpu_op_s_swappc_b64:
-                case amdgpu_gfx908_op_S_SETPC_B64:
-                case amdgpu_gfx908_op_S_SWAPPC_B64:
-                case amdgpu_gfx90a_op_S_SETPC_B64:
-                case amdgpu_gfx90a_op_S_SWAPPC_B64:
                     return false;
+#if 1
+                /*
+                 * These amdgpu instructions are placed here to allow
+                 * flow control processing with the new optional lack
+                 * of Semantic Evaluation feature for amdgpu, and a few
+                 * other odd cases due to instruction classification issues.
+                 *
+                 * This temporary fix allows almost full dyninst CFG
+                 * until we have amdgpu semantic information.
+                 * Once semantic evaluation is received & implemented
+                 * for amdgpu architectures, this will no longer be needed;
+                 * then GPU instructions will be processed as e_ "instructions".
+                 */
+#include "amdgpu_callinsn_table.h"
```

Comment:
I know there are lots of places in Dyninst where we include source files and headers in the middle of blocks, but let's not perpetuate that terrible coding style. If these cases have a special meaning, we should call them out in a function.

Reply:
Those are machine generated files from the database dumps.
This way, as changes happen and new AMD GPU architectures are added, the machine
generated changes stay in machine generated files, not in regular files
that need to have those changes  hand edited into.